src/data/queries/mediaVideo.ts

```diff
@@ -0,0 +1,46 @@
+import { QueryData, QueryFormatter, QueryParams } from 'next-drupal-query'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+import { MediaVideo } from '@/types/formatted/media'
+import { DrupalMediaVideo } from '@/types/drupal/media'
+import { drupalClient } from '@/lib/drupal/drupalClient'
+
+// Define the query params for fetching node--media_video.
+export const params: QueryParams<null> = () => {
+  return new DrupalJsonApiParams().addInclude(['video'])
```

Comment:
this include fails for me, it may not be necessary for videos to include anything